src/sensors/distant.cpp

```diff
@@ -263,12 +256,14 @@ MTS_VARIANT class DistantSensor final : public Sensor<Float, Spectrum> {
 
```

Comment:
The `to_string()` method should not be neglected. You should vary it based on `m_origin_type`.